README.md

```diff
@@ -18,10 +36,80 @@ $$ T(n) =
     \end{cases}
 $$
 
+**Answer:**
+
+Attempt to use the Master theorem method. I am following this website's Master Theorem method. https://www.programiz.com/dsa/master-theorem
+
+Therom method = T(n) = $aT\left(\frac{n}{b}\right) + f(n)$
+
+We have: 
+
+$T(n)$ = $13T\left(\frac{n}{13}\right) + 5$
+
+a = 13 
+
+b = 13
+
+f(n) = 5
+
+Step 2: Need to solve $log_{a} b$
+
+$log_{13} 13$ = 1 
+
+Step 3: Compare the f(n) to $log_{a} b$
+
+f(n) = 5 
+
+$n^{log_{a} b}$ = $n^1$ = _n_
+
+Since f(n) is $O(n^{1 - &#949;})$, the Master theorm states $T(n) = Theta(n^{log_{a} b})$
+
+Which will give us T(n) = $Theta(n^1)$ = $Theta(n)$ 
+
+Answer = $\theta(n)$
+
+
 3.
 $$ T(n) =
     \begin{cases}
         1 & n \leq 1\\
         13 T\left(\frac{n}{13}\right) + 2n & n > 1
     \end{cases}
 $$
+
+**Answer:**
+
+Use the Master theorem again. 
+
+a = 13
+
+b = 13
+
+f(n) = 2n
+
+Step 2: solve $log_{a} b$
+
+$log_{13} 13$ = 1 
+
+Step 3: Compare f(n) to $log_{a} b$ 
+
+f(n) = 2n 
+
+$n^{log_{a} b}$ = $n^1$ = _n_
+
+f(n) is in the same order as log 
+
+Step 4: Solve 
+
+$T(n) = (n^{log_{a} b} * log n)$
+
+Since $n^{log_{13} 13} = n$ 
+
+$T(n) = \theta(n log n)$ is the answer 
```

Comment:
Notation still incorrect -- a T(n) expression cannot be equal to an asymptotic bound.